client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -26,87 +26,69 @@
 public class RssSparkConfig {
 
   public static final String SPARK_RSS_CONFIG_PREFIX = "spark.";
-  public static final String RSS_PARTITION_NUM_PER_RANGE =
-      SPARK_RSS_CONFIG_PREFIX + RssClientConfig.RSS_PARTITION_NUM_PER_RANGE;
+  public static final String RSS_PARTITION_NUM_PER_RANGE = prefix(RssClientConfig.RSS_PARTITION_NUM_PER_RANGE);
```

Comment:
```
SPARK_RSS_CONFIG_PREFIX + RSS_XXX_XXX
```
is enough simple. When we use the method `prefix`, we need look at the method concrete implement. I don't think the pr improve the readability. And is there any other system to do similar code optimization? Hadoop use the same style as ours.